src/components/Card/Card.js

```diff
@@ -191,7 +174,13 @@ const Card = ({ mentor, onFavMentor, isFav }) => {
         <div className="wave" />
         <div className="channels">
           {availability ? (
-            <div className="channel-inner">{channelsList(channels)}</div>
+            <div className="channel-inner">
+              {auth.isAuthenticated() ? (
+                <LoggedInApply mentor={mentor} />
```

Comment:
Can be a one component for both loggedin and anonymous because the UI is very similar (apply button with a tooltip and `onClick` callback)